ui/imports/shared/popups/SendModal.qml

```diff
@@ -64,23 +64,24 @@ ModalPopup {
                         stack.uuid)
         }
 
-        if(!success){
-            //% "Invalid transaction parameters"
-            sendingError.text = qsTrId("invalid-transaction-parameters")
-            sendingError.open()
-        } else {
-            // TODO remove this else once the thread and connection are back
-            stack.currentGroup.isPending = false
-            //% "Transaction pending..."
-            Global.toastMessage.title = qsTrId("ens-transaction-pending")
-            Global.toastMessage.source = Style.svg("loading")
-            Global.toastMessage.iconColor = Style.current.primary
-            Global.toastMessage.iconRotates = true
-            // Refactor this
-            // Global.toastMessage.link = `${walletModel.utilsView.etherscanLink}/${response.result}`
-            Global.toastMessage.open()
-            root.close()
-        }
+        // Till the method is moved to thread this is handled by a signal to which connection is made in the end of the file
+//        if(!success){
+//            //% "Invalid transaction parameters"
+//            sendingError.text = qsTrId("invalid-transaction-parameters")
+//            sendingError.open()
+//        } else {
+//            // TODO remove this else once the thread and connection are back
+//            stack.currentGroup.isPending = false
+//            //% "Transaction pending..."
+//            Global.toastMessage.title = qsTrId("ens-transaction-pending")
+//            Global.toastMessage.source = Style.svg("loading")
+//            Global.toastMessage.iconColor = Style.current.primary
+//            Global.toastMessage.iconRotates = true
+//            // Refactor this
+//            // Global.toastMessage.link = `${walletModel.utilsView.etherscanLink}/${response.result}`
+//            Global.toastMessage.open()
+//            root.close()
+//        }
```

Comment:
The else can be deleted. I had added it because I didn't use a signal yet, but since you added the signal, the else can go. Only the `if` will be added back once we use the thread.

Reply:
Sure :)